packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/openapi/openapi-validation.test.ts

```diff
@@ -0,0 +1,482 @@
+import { AddressInfo } from "net";
+import http from "http";
+import fs from "fs-extra";
+import path from "path";
+import test, { Test } from "tape-promise/tape";
+import { v4 as uuidv4 } from "uuid";
+import express from "express";
+import bodyParser from "body-parser";
+import {
+  Containers,
+  FabricTestLedgerV1,
+  pruneDockerAllIfGithubAction,
+} from "@hyperledger/cactus-test-tooling";
+import {
+  IListenOptions,
+  LogLevelDesc,
+  Servers,
+} from "@hyperledger/cactus-common";
+import { PluginRegistry } from "@hyperledger/cactus-core";
+import {
+  ChainCodeProgrammingLanguage,
+  DefaultEventHandlerStrategy,
+  DeployContractV1Request,
+  FabricContractInvocationType,
+  FileBase64,
+  PluginLedgerConnectorFabric,
+  RunTransactionRequest,
+} from "../../../../main/typescript/public-api";
+import { DefaultApi as FabricApi } from "../../../../main/typescript/public-api";
+import { IPluginLedgerConnectorFabricOptions } from "../../../../main/typescript/plugin-ledger-connector-fabric";
+import { DiscoveryOptions } from "fabric-network";
+import { PluginKeychainMemory } from "@hyperledger/cactus-plugin-keychain-memory";
+import { Configuration } from "@hyperledger/cactus-core-api";
+
+import { installOpenapiValidationMiddleware } from "@hyperledger/cactus-core";
+import OAS from "../../../../main/json/openapi.json";
+
+const testCase = "deploys Fabric 2.x contract from typescript source";
+const logLevel: LogLevelDesc = "TRACE";
+
+test("BEFORE " + testCase, async (t: Test) => {
+  const pruning = pruneDockerAllIfGithubAction({ logLevel });
+  await t.doesNotReject(pruning, "Pruning didn't throw OK");
+  t.end();
+});
+
+test(testCase, async (t: Test) => {
+  const channelId = "mychannel";
+  const channelName = channelId;
+
+  test.onFailure(async () => {
+    await Containers.logDiagnostics({ logLevel });
+  });
+
+  const ledger = new FabricTestLedgerV1({
+    emitContainerLogs: true,
+    publishAllPorts: true,
+    imageName: "ghcr.io/hyperledger/cactus-fabric2-all-in-one",
+    imageVersion: "2021-04-20-nodejs",
```

Comment:
@elenaizaguirre Please update the image to the new one (see other tests on the main branch)

Reply:
Done!